src/PhpSpreadsheet/Calculation/Web.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheet\Calculation;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\ConnectException;
+
+class Web
+{
+    protected static $client;
+
+    /**
+     * Override the default webclient.
+     * Used for test purposes.
+     *
+     * @param Client $client
+     */
+    public static function setClient(Client $client)
```

Comment:
This should live in `\PhpOffice\PhpSpreadsheet\Settings`, like the similar `setCache/getCache`, so settings are centralized and could _maybe_ later be transformed into a PSR-11.